src/components/ActionRadio/ActionRadio.vue

```diff
@@ -0,0 +1,244 @@
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+This component is made to be used inside of the [Actions](#Actions) component slots.
+Usually, you will provide a name prop to bind the radio together.
+So that only one of each name set can be selected at the same time.
+
+```vue
+	<Actions>
+		<ActionRadio @change="alert('(un)checked !')" name="uniqueId">First choice</ActionRadio>
+		<ActionRadio value="second" name="uniqueId" @check="alert('checked !')">Second choice</ActionRadio>
+		<ActionRadio :checked="true" name="uniqueId" @uncheck="alert('unchecked !')">Third choice (checked)</ActionRadio>
+		<ActionRadio :disabled="true" name="uniqueId" @change="alert('(un)checked !')">Second choice (disabled)</ActionRadio>
+	</Actions>
+```
+</docs>
+
+<template>
+	<li :class="{ 'action--disabled': disabled }">
+		<span class="action-radio">
+			<input :id="id" ref="radio"
+				:disabled="disabled" :checked="checked"
+				:name="name" :value="value"
+				:class="{ focusable: isFocusable }"
+				type="radio" class="radio action-radio__radio"
+				@keydown.enter.exact.prevent="toggleInput" @change="onChange">
+			<label ref="label" :for="id" class="action-radio__label">{{ text }}</label>
+
+			<!-- fake slot to gather inner text -->
+			<slot v-if="false" />
+		</span>
+	</li>
+</template>
+
+<script>
+import ActionGlobalMixin from 'Mixins/actionGlobal'
+import GenRandomId from 'Utils/GenRandomId'
+import GetParent from 'Utils/GetParent'
+import GetChildren from 'Utils/GetChildren'
+
+export default {
+	name: 'ActionRadio',
+
+	mixins: [ActionGlobalMixin],
+
+	props: {
+		/**
+		 * id attribute of the radio element
+		 */
+		id: {
+			type: String,
+			default: () => 'action-' + GenRandomId(),
+			validator: id => id.trim() !== ''
+		},
+
+		/**
+		 * checked state of the the radio element
+		 */
+		checked: {
+			type: Boolean,
+			default: false
+		},
+
+		/**
+		 * Define if this radio is part of a set.
+		 * Checking the radio will disable all the
+		 * others with the same name.
+		 */
+		name: {
+			type: String,
+			required: true
+		},
+
+		/**
+		 * value of the radio input
+		 */
+		value: {
+			type: [String, Number],
+			default: ''
+		},
+
+		/**
+		 * disabled state of the radio element
+		 */
+		disabled: {
+			type: Boolean,
+			default: false
+		}
+	},
+
+	computed: {
+		/**
+		 * determines if the action is focusable
+		 */
+		isFocusable: function() {
+			return !this.disabled
+		}
+	},
+
+	methods: {
+		toggleInput(event) {
+			// by clicking we also trigger the change event
+			this.$refs.label.click()
+		},
+		onChange(event) {
+			/**
+			 * Emitted when the radio state is changed
+			 * @type {Event}
+			 */
+			this.$emit('change', event)
+
+			if (this.$refs.radio.checked) {
+				/**
+				 * Emitted when the radio is checked
+				 * @type {Event}
+				 */
+				this.$emit('check', true)
+
+				// if part of a set, firing events on all the other
+				if (this.name) {
```

Comment:
```suggestion
				if (!this.name) {
```

and early return to lower the number of indentations